shell/edit/provisioning.cattle.io.cluster/tabs/registries/RegistryConfigs.vue

```diff
@@ -87,9 +68,27 @@ export default {
     window.z = this;
   },
 
-  // created() {
-  //   set(this.value, 'spec.rkeConfig.registries.configs', {});
```

Comment:
It looks like we're no longer initializing `spec.rkeConfig.registries.configs` to { -- was this intentional? The component appears to function without this line; I tried creating and editing clusters with custom registry authentication (which this component controls) and didn't see bugs.